fast-list.js

```diff
@@ -258,6 +299,11 @@ FastList.prototype = {
       );
     }
```

Comment:
it used to make sure all reusable nodes where inserted before we start scrolling.
I think we do this in phase 2 now, or at least we should :)